.github/workflows/rules/md104.js

```diff
@@ -9,13 +9,18 @@ module.exports = {
       return token.type === "inline";
     })) {
       var actual_lines = inline.content.split("\n");
+      var hugo_shortcode = false;
       actual_lines.forEach((line, index, arr) => {
+		if (line.startsWith("{{"))
+		      hugo_shortcode = true;
+		if (line.endsWith("}}"))
+		      hugo_shortcode = false;
```

Comment:
I would aim to cover the case where there is a single shortcode line, i.e. starting with `{{` and ending with `}}`. So what I would do is move this check (`line.endsWith`) some place after the `Array.from(line)...` part. Wouldn't this work?

Reply:
That works great, done.